mupen64plus-core/upstream/src/api/frontend.c

```diff
@@ -69,6 +69,8 @@ EXPORT m64p_error CALL CoreStartup(int APIVersion, const char *ConfigPath, const
     SetDebugCallback(DebugCallback, Context);
     SetStateCallback(StateCallback, Context2);
 
+    g_EmuModeInitiated = 0;
```

Comment:
Make a pull request here for this change:
https://github.com/mupen64plus-ae/mupen64plus-core

Reply:
Well, do it for all core changes.